old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/04-intro-CRS-projection.Rmd

```diff
@@ -81,20 +82,20 @@ downloadable R script of the entire lesson, available in the footer of each less
 of a lesson series introducing
 [spatial data and data management in `R` ](https://www.neonscience.org/tutorial/URL).
```

Comment:
@donal-at-NEON: I didn't update this tutorial link because it references THIS tutorial series. I'm not sure what the URL will be once it's published (although you could have told me and I forgot!)

Reply:
Sure! What's the best way to make a note for this?

Reply:
hah good question!
How about:
##### UPDATE BEFORE PUBLISHING #####